utils/joe/files/joerc

```diff
@@ -7,46 +6,55 @@
 -assume_color
 -assume_256color
 -guess_non_utf8
+-guess_utf16
 -guess_crlf
 -guess_indent
 -menu_above
 -transpose
--joe_state
--restore
--mouse
+ -joe_state
```

Comment:
This line ``` + -joe_state``` looks suspiciousyl like an accidental whitespace problem. Or is the leading space intentional?